Pyrado/pyrado/algorithms/step_based/value_based.py

```diff
@@ -160,9 +162,20 @@ def memory(self) -> ReplayMemory:
     def step(self, snapshot_mode: str, meta_info: dict = None):
         if self._memory.isempty:
             # Warm-up phase
-            print_cbt_once("Collecting samples until replay memory if full.", "w")
+            print_cbt_once(f"Empty replay memory, collecting {self.num_init_memory_steps} samples.", "w")
             # Sample steps and store them in the replay memory
-            ros = self.sampler_init.sample()
+            if self._fill_with_init_sampler:
+                ros = self.sampler_init.sample()
+                self._fill_with_init_sampler = False
```

Comment:
it feels like an unintuitive design decision that the behavior changes automatically after the first reset. let's not do it like this, but rather make sure that the user has to trigger this behavior manually, e.g. by setting a hyper-parameter

Reply:
What do you think about e487cac5a8ef7697a54e54c1b20e09ce3831cbbb?